src/analyses/does_remove_const.cpp

```diff
@@ -13,19 +13,15 @@ Author: Diffblue Ltd.
 
 #include <goto-programs/goto_program.h>
 
-#include <util/base_type.h>
 #include <util/pointer_expr.h>
 #include <util/std_code.h>
 
 /// A naive analysis to look for casts that remove const-ness from pointers.
 /// \param goto_program: the goto program to check
 /// \param ns: the namespace of the goto program (used for checking type
 ///   equality)
-does_remove_constt::does_remove_constt(
-  const goto_programt &goto_program,
-  const namespacet &ns):
-    goto_program(goto_program),
-    ns(ns)
+does_remove_constt::does_remove_constt(const goto_programt &goto_program)
+  : goto_program(goto_program)
```

Comment:
Thank you for adding this commit. The documentation above, however, also needs to be updated (removing lines 21 and 22) to keep doxygen happy.